helpdesk/views/kb.py

```diff
@@ -30,7 +30,7 @@ def category(request, slug, iframe=False):
     category = get_object_or_404(KBCategory, slug__iexact=slug)
     if not user.huser_from_request(request).can_access_kbcategory(category):
         raise Http404
-    items = category.kbitem_set.all()
+    items = category.kbitem_set.filter(enabled=True)
```

Comment:
https://github.com/auto-mat/django-helpdesk/blob/teams/helpdesk/forms.py#L187

this line needs to be updated to filter out disabled items too.

Reply:
Do you mean `KBCategory` with `public=False`?

Reply:
No, I mean that in the Ticket form, you can choose the kbitem that the ticket belongs to. That choice field (which I linked to) should not include kbitems which are not enabled.

Reply:
@timthelion Fixed.